### ISSUE TITLE
New release needed: AverageOutput is broken in current "latest" release

### ISSUE THREAD

**Issue by jamshark70** [supercollider/sc3-plugins#277]
A user found this bug:

```
(
a = {
	var sig = Line.kr(100, 0, 10, doneAction: 2),
	avg = AverageOutput.kr(sig);
	avg.poll(2);
	Silent.ar(1)
}.play;
)

UGen(AverageOutput): 99.9855
UGen(AverageOutput): 637.489
UGen(AverageOutput): 701.58
UGen(AverageOutput): 736.966
UGen(AverageOutput): 760.715
```

The Line is decreasing so the average should decrease too, but it's plainly rising.

I find "brianlheim fix unsequenced modification of value c87cdb4 on Jan 12, 2019." I'm using this update (built myself from source) and I can't reproduce the bug.

The "latest release" is 3.10.0, packaged in November 2018.

So there are known code fixes that users are not currently downloading.

**Comment by dyfer** [comment]
AFAIK 3.11 release is in the works. Recently Brian merged updates from 3.10 branch into master, so I'd guess this fix should be included.

**Comment by dyfer** [comment]
I think this was solved in 3.11. If not, feel free to reopen.
